src/plugins/auto/auto_schedule.cpp

```diff
@@ -459,15 +567,21 @@ bool AutoSchedule::ScheduleToWorkerInferRequest(IE::Task inferPipelineTask, Devi
         }
         devices.push_back(_loadContext[ACTUALDEVICE].deviceInfo);
     } else {
-        // _acceleratorDevice could be the same as _cpuDevice, such as AUTO:CPU
```

Comment:
the preferred device code path need to be updated with ctput also

Reply:
Updated.

Reply:
kind of redundant with line#731, probably we just need to avoid waitActual in ctput mode.
btw, with remote blob and runtime fall back enabled, we should break out the runtime fallback loop because the fallback device is never able to deal with other device's remote memory. Is this been implemented?